src/css/objects/_icon.scss

```diff
@@ -0,0 +1,24 @@
+.icon {
+  display: block;
```

Comment:
Icons usually do not have display as block. Also, looking at the rules below it seems like you are overriding the display property every time. Any reason for `block`?